### PR TITLE
Cleanup duplicates

### PR DESCRIPTION
**Pull request by nstephenh**
I wrote a script to check for all the duplicate entries. Here's the results. I'll re-run the script and re-save all the files tomorrow, as I think my branch was a bit old.
The things I have left to check before undrafting are:
- [x] ensure I'm not making [links to links](https://github.com/nstephenh/BSCopy/issues/10)
- [x] ensure no constraints or modifiers were dropped
- [x] check all the dark blessings and figure out what the current status of those is, unmerge them by hand.
- [x] manually move a few special rules, namely open-topped.

### CODE REVIEW

**Comment by The4D6** [comment]
The scope of this PR has gotten too large.

Could we please try to split this down either in to two PRS (one for spaces, one for dupes), or break it down file by file?

It's hard to review given that you appear to have made edits to every file in the game, and interpreting the references just in GitHub is very difficult. Even when looked at in the NR editor it is not immediately apparent which version has been kept and why.

**Comment by nstephenh** [comment]
> Could we please try to split this down either in to two PRS (one for spaces, one for dupes), or break it down file by file?
Sorry, I forgot to check out a new branch. This should now be based on master, and the spaces one should be merged already.

> It's hard to review given that you appear to have made edits to every file in the game, and interpreting the references just in GitHub is very difficult. Even when looked at in the NR editor it is not immediately apparent which version has been kept and why.
I somewhat agree, I'll be checking this in pycharm on the raw XML.
The prioritization I use is "GST, cat library, first appearance" in that order.

**Comment by nstephenh** [comment]
Confirmed I'm definitely deleting modifiers so I'll need to rebranch and fix that.

**Comment by nstephenh** [comment]
I think this is clean enough for review at this point, let me know if you have any questions or spot anything of concern.

File: Daemon Library.cat [hunk 1]
```diff
@@ -21,7 +21,7 @@ A unit which includes models with this special rule without a specific Ætheric
           </constraints>
           <rules>
             <rule id="4438-874e-901d-f86" name="Encroaching Ruin" publicationId="8775-88f5-cfdd-24f6" page="3" hidden="false">
-              <description>When a unit entirely composed of models with this special rule and the Daemon Unit Type fails a Morale check, it only suffers 1 automatic Wound with no Saves of any kind allowed instead of D3 Wounds as specified in the Daemon Unit Type. In addition, models with this special rule gain the Move Through Cover special rule.</description>
+              <description>When a unit entirely composed of models with this special rule and the Daemon Unit Type fails a Morale check, it only suffers 1 automatic Wound with no Saves of any kind allowed instead of D3 Wounds as specified in the Daemon Unit Type. In addition, models with this special rule gain the Move Through Cover special rule.</description>
```
Comment (The4D6) [inline Daemon Library.cat:24]: I thought we'd move nonbreaking spaces to their own PR.

If so, that should be there

**Review by The4D6 — CHANGES_REQUESTED** [review]
Looks good, just needs the one spacing issue moved out in to it's own PR as previously discussed, once that's done I'm happy with it

File: Daemon Library.cat (hunk 1 above)
Comment (nstephenh) [reply Daemon Library.cat:24]: NBSP was merged already. This change appears to be " " to "nbsp". These are ones I recopied from the book to ensure I had the most recent wording, and that's why they have nbsp in them. Fixing.
